2023-10-21 20:53:40,055 INFO    StreamThr :2540 [internal.py:wandb_internal():86] W&B internal server running at pid: 2540, started at: 2023-10-21 20:53:40.049083
2023-10-21 20:53:40,058 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status
2023-10-21 20:53:40,088 INFO    WriterThread:2540 [datastore.py:open_for_write():85] open: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\run-lph8yyg7.wandb
2023-10-21 20:53:40,089 DEBUG   SenderThread:2540 [sender.py:send():379] send: header
2023-10-21 20:53:40,256 DEBUG   SenderThread:2540 [sender.py:send():379] send: run
2023-10-21 20:53:40,998 INFO    SenderThread:2540 [dir_watcher.py:__init__():211] watching files in: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files
2023-10-21 20:53:40,999 INFO    SenderThread:2540 [sender.py:_start_run_threads():1121] run started: lph8yyg7 with start time 1697914420.056081
2023-10-21 20:53:41,015 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: check_version
2023-10-21 20:53:41,019 DEBUG   SenderThread:2540 [sender.py:send_request():406] send_request: check_version
2023-10-21 20:53:41,174 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: run_start
2023-10-21 20:53:42,062 DEBUG   HandlerThread:2540 [system_info.py:__init__():32] System info init
2023-10-21 20:53:42,063 DEBUG   HandlerThread:2540 [system_info.py:__init__():47] System info init done
2023-10-21 20:53:42,063 INFO    HandlerThread:2540 [system_monitor.py:start():181] Starting system monitor
2023-10-21 20:53:42,064 INFO    SystemMonitor:2540 [system_monitor.py:_start():145] Starting system asset monitoring threads
2023-10-21 20:53:42,064 INFO    HandlerThread:2540 [system_monitor.py:probe():201] Collecting system info
2023-10-21 20:53:42,077 INFO    SystemMonitor:2540 [interfaces.py:start():190] Started cpu monitoring
2023-10-21 20:53:42,079 INFO    SystemMonitor:2540 [interfaces.py:start():190] Started disk monitoring
2023-10-21 20:53:42,082 INFO    SystemMonitor:2540 [interfaces.py:start():190] Started gpu monitoring
2023-10-21 20:53:42,098 INFO    SystemMonitor:2540 [interfaces.py:start():190] Started memory monitoring
2023-10-21 20:53:42,113 DEBUG   HandlerThread:2540 [system_info.py:probe():196] Probing system
2023-10-21 20:53:42,142 INFO    SystemMonitor:2540 [interfaces.py:start():190] Started network monitoring
2023-10-21 20:53:42,180 DEBUG   HandlerThread:2540 [system_info.py:_probe_git():181] Probing git
2023-10-21 20:53:42,180 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:53:42,553 DEBUG   HandlerThread:2540 [system_info.py:_probe_git():189] Probing git done
2023-10-21 20:53:42,553 DEBUG   HandlerThread:2540 [system_info.py:probe():244] Probing system done
2023-10-21 20:53:42,553 DEBUG   HandlerThread:2540 [system_monitor.py:probe():210] {'os': 'Windows-10-10.0.19045-SP0', 'python': '3.10.0', 'heartbeatAt': '2023-10-21T18:53:42.142097', 'startedAt': '2023-10-21T18:53:40.015077', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '<python with no main file>', 'codePathLocal': None, 'git': {'remote': 'https://github.com/marlens123/pond_segmentation.git', 'commit': 'd79147d1f703af9da2b1b9e8cdf3d450fe5fdb9f'}, 'email': 'marlena@itdex.de', 'root': 'E:/polar/pond_segmentation', 'host': 'DEXSURFACE', 'username': 'marle', 'executable': 'c:\\Users\\marle\\miniconda3\\envs\\extended\\python.exe', 'cpu_count': 2, 'cpu_count_logical': 4, 'cpu_freq': {'current': 2607.0, 'min': 0.0, 'max': 2808.0}, 'cpu_freq_per_core': [{'current': 2607.0, 'min': 0.0, 'max': 2808.0}], 'disk': {'/': {'total': 59.462886810302734, 'used': 47.120208740234375}}, 'gpu': 'NVIDIA GeForce GPU', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce GPU', 'memory_total': 1073741824}], 'memory': {'total': 15.919273376464844}}
2023-10-21 20:53:42,554 INFO    HandlerThread:2540 [system_monitor.py:probe():211] Finished collecting system info
2023-10-21 20:53:42,554 INFO    HandlerThread:2540 [system_monitor.py:probe():214] Publishing system info
2023-10-21 20:53:42,554 DEBUG   HandlerThread:2540 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2023-10-21 20:53:42,563 DEBUG   HandlerThread:2540 [system_info.py:_save_pip():68] Saving pip packages done
2023-10-21 20:53:42,564 DEBUG   HandlerThread:2540 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2023-10-21 20:53:42,589 ERROR   HandlerThread:2540 [system_info.py:_save_conda():86] Error saving conda packages: [WinError 2] Das System kann die angegebene Datei nicht finden
Traceback (most recent call last):
  File "c:\Users\marle\miniconda3\envs\extended\lib\site-packages\wandb\sdk\internal\system\system_info.py", line 82, in _save_conda
    subprocess.call(
  File "c:\Users\marle\miniconda3\envs\extended\lib\subprocess.py", line 345, in call
    with Popen(*popenargs, **kwargs) as p:
  File "c:\Users\marle\miniconda3\envs\extended\lib\subprocess.py", line 966, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "c:\Users\marle\miniconda3\envs\extended\lib\subprocess.py", line 1435, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] Das System kann die angegebene Datei nicht finden
2023-10-21 20:53:42,593 DEBUG   HandlerThread:2540 [system_info.py:_save_conda():87] Saving conda packages done
2023-10-21 20:53:42,597 INFO    HandlerThread:2540 [system_monitor.py:probe():216] Finished publishing system info
2023-10-21 20:53:42,649 DEBUG   SenderThread:2540 [sender.py:send():379] send: files
2023-10-21 20:53:42,649 INFO    SenderThread:2540 [sender.py:_save_file():1375] saving file wandb-metadata.json with policy now
2023-10-21 20:53:43,013 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: stop_status
2023-10-21 20:53:43,015 DEBUG   SenderThread:2540 [sender.py:send_request():406] send_request: stop_status
2023-10-21 20:53:43,043 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_created():271] file/dir created: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\conda-environment.yaml
2023-10-21 20:53:43,046 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_created():271] file/dir created: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\requirements.txt
2023-10-21 20:53:43,047 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_created():271] file/dir created: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\wandb-metadata.json
2023-10-21 20:53:43,063 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: internal_messages
2023-10-21 20:53:43,381 DEBUG   SenderThread:2540 [sender.py:send():379] send: telemetry
2023-10-21 20:53:43,732 DEBUG   SenderThread:2540 [sender.py:send():379] send: telemetry
2023-10-21 20:53:43,750 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:53:43,752 INFO    wandb-upload_0:2540 [upload_job.py:push():131] Uploaded file C:\Users\marle\AppData\Local\Temp\tmpdl6h20f2wandb\zrkpmuby-wandb-metadata.json
2023-10-21 20:53:43,755 DEBUG   SenderThread:2540 [sender.py:send():379] send: telemetry
2023-10-21 20:53:43,756 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:53:43,758 WARNING SenderThread:2540 [sender.py:send_metric():1326] Seen metric with glob (shouldn't happen)
2023-10-21 20:53:44,044 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_created():271] file/dir created: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\output.log
2023-10-21 20:53:44,228 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:53:44,229 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:53:44,425 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:53:45,779 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:53:46,058 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\output.log
2023-10-21 20:53:46,439 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:53:46,439 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:53:46,696 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:53:48,709 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:53:48,709 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:53:48,910 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:53:50,826 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:53:50,922 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:53:50,923 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:53:51,109 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:53:53,113 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:53:53,114 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:53:53,297 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:53:55,301 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:53:55,302 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:53:55,489 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:53:55,877 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:53:57,507 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:53:57,508 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:53:57,697 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:53:58,027 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: stop_status
2023-10-21 20:53:58,028 DEBUG   SenderThread:2540 [sender.py:send_request():406] send_request: stop_status
2023-10-21 20:53:58,077 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: internal_messages
2023-10-21 20:53:59,711 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:53:59,711 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:53:59,896 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:01,320 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:54:01,904 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:01,906 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:02,091 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:04,109 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:04,109 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:04,295 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:06,286 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\output.log
2023-10-21 20:54:06,370 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:06,371 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:06,471 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:54:06,572 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:08,432 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\output.log
2023-10-21 20:54:08,616 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:08,616 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:08,847 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:10,889 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:10,890 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:11,136 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:11,745 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:54:12,898 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\config.yaml
2023-10-21 20:54:13,104 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: stop_status
2023-10-21 20:54:13,105 DEBUG   SenderThread:2540 [sender.py:send_request():406] send_request: stop_status
2023-10-21 20:54:13,197 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: internal_messages
2023-10-21 20:54:13,198 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:13,200 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:13,688 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:16,612 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:17,431 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:54:18,660 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:18,660 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:19,068 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:21,105 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:21,105 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:21,357 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:22,508 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:54:22,923 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\output.log
2023-10-21 20:54:23,371 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:23,371 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:23,623 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:25,642 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:25,643 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:25,892 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:27,948 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:27,950 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:54:27,950 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:28,162 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: stop_status
2023-10-21 20:54:28,164 DEBUG   SenderThread:2540 [sender.py:send_request():406] send_request: stop_status
2023-10-21 20:54:28,244 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: internal_messages
2023-10-21 20:54:28,271 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:30,315 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:30,316 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:30,552 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:32,590 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:32,591 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:32,889 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:33,638 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:54:34,941 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:34,941 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:35,200 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:36,894 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\output.log
2023-10-21 20:54:37,229 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:37,236 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:37,518 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:38,939 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:54:38,941 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\output.log
2023-10-21 20:54:39,545 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:39,545 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:39,747 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:41,756 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:41,756 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:41,995 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:42,198 DEBUG   SystemMonitor:2540 [system_monitor.py:_start():159] Starting system metrics aggregation loop
2023-10-21 20:54:42,205 DEBUG   SenderThread:2540 [sender.py:send():379] send: stats
2023-10-21 20:54:43,218 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: stop_status
2023-10-21 20:54:43,220 DEBUG   SenderThread:2540 [sender.py:send_request():406] send_request: stop_status
2023-10-21 20:54:43,547 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: internal_messages
2023-10-21 20:54:44,064 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:44,066 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:44,381 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:44,557 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:54:46,408 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:46,408 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:46,676 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:48,713 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:48,715 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:48,960 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:49,594 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:54:51,776 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:52,222 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\output.log
2023-10-21 20:54:53,807 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:53,807 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:54,084 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:55,566 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:54:56,117 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:56,117 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:56,439 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:54:58,248 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: stop_status
2023-10-21 20:54:58,249 DEBUG   SenderThread:2540 [sender.py:send_request():406] send_request: stop_status
2023-10-21 20:54:58,307 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: internal_messages
2023-10-21 20:54:58,467 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:58,468 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:54:58,728 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:00,745 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:00,746 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:01,131 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:01,475 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:55:03,346 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:03,346 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:03,557 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:05,601 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:05,602 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:05,875 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:06,574 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\output.log
2023-10-21 20:55:07,132 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:55:07,902 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:07,907 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:08,148 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:08,732 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\output.log
2023-10-21 20:55:10,186 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:10,187 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:10,584 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:12,431 DEBUG   SenderThread:2540 [sender.py:send():379] send: stats
2023-10-21 20:55:12,434 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:55:12,751 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:12,751 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:13,044 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:13,278 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: stop_status
2023-10-21 20:55:13,279 DEBUG   SenderThread:2540 [sender.py:send_request():406] send_request: stop_status
2023-10-21 20:55:13,322 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: internal_messages
2023-10-21 20:55:15,148 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:15,149 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:15,519 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:17,565 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:17,566 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:17,781 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:55:17,851 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:18,936 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\output.log
2023-10-21 20:55:19,890 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:19,896 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:20,169 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:21,050 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\output.log
2023-10-21 20:55:22,223 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:22,224 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:22,439 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:23,212 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:55:24,452 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:24,452 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:24,672 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:27,244 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:28,304 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:55:28,306 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: stop_status
2023-10-21 20:55:28,306 DEBUG   SenderThread:2540 [sender.py:send_request():406] send_request: stop_status
2023-10-21 20:55:28,334 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: internal_messages
2023-10-21 20:55:29,341 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:29,342 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:29,701 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:31,791 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\output.log
2023-10-21 20:55:31,793 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:31,937 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:32,172 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:33,610 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:55:33,899 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\output.log
2023-10-21 20:55:34,217 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:34,217 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:34,455 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:36,520 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:36,520 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:36,770 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:38,884 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:38,884 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:39,216 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:39,376 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:55:41,285 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:41,285 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:41,573 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:42,324 DEBUG   SenderThread:2540 [sender.py:send():379] send: stats
2023-10-21 20:55:43,334 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: stop_status
2023-10-21 20:55:43,336 DEBUG   SenderThread:2540 [sender.py:send_request():406] send_request: stop_status
2023-10-21 20:55:43,342 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: internal_messages
2023-10-21 20:55:43,602 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:43,626 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:43,894 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:44,770 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:55:45,909 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:45,909 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:46,229 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:48,246 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:48,246 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:48,278 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\output.log
2023-10-21 20:55:48,486 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:50,178 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:55:50,494 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:50,494 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:50,712 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:52,738 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:52,738 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:52,990 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:55,018 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:55,018 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:55,272 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:55,322 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:55:57,275 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:57,275 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:57,490 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:55:57,828 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\output.log
2023-10-21 20:55:58,352 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: stop_status
2023-10-21 20:55:58,352 DEBUG   SenderThread:2540 [sender.py:send_request():406] send_request: stop_status
2023-10-21 20:55:58,356 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: internal_messages
2023-10-21 20:55:59,506 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:59,506 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:55:59,692 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:56:00,492 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:56:02,048 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\output.log
2023-10-21 20:56:02,093 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:56:04,104 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:56:04,104 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:56:04,308 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:56:05,690 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:56:06,321 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:56:06,321 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:56:06,508 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:56:06,877 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: partial_history
2023-10-21 20:56:06,897 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:56:06,897 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:56:06,897 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:56:06,897 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:56:06,897 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:56:06,897 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:56:06,897 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:56:06,897 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:56:06,897 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:56:06,897 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:56:06,897 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:56:06,901 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:56:06,901 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:56:06,901 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:56:06,901 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:56:06,901 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:56:06,901 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:56:06,905 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:56:06,905 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:56:06,905 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:56:06,905 DEBUG   SenderThread:2540 [sender.py:send():379] send: metric
2023-10-21 20:56:06,909 DEBUG   SenderThread:2540 [sender.py:send():379] send: history
2023-10-21 20:56:06,909 DEBUG   SenderThread:2540 [sender.py:send_request():406] send_request: summary_record
2023-10-21 20:56:06,917 INFO    SenderThread:2540 [sender.py:_save_file():1375] saving file wandb-summary.json with policy end
2023-10-21 20:56:07,125 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_created():271] file/dir created: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\wandb-summary.json
2023-10-21 20:56:08,369 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\output.log
2023-10-21 20:56:08,562 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:56:08,562 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:56:08,847 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:56:10,507 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\output.log
2023-10-21 20:56:10,878 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:56:10,882 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:56:10,941 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:56:11,160 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:56:12,382 DEBUG   SenderThread:2540 [sender.py:send():379] send: stats
2023-10-21 20:56:13,203 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:56:13,211 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:56:13,351 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: stop_status
2023-10-21 20:56:13,351 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: internal_messages
2023-10-21 20:56:13,359 DEBUG   SenderThread:2540 [sender.py:send_request():406] send_request: stop_status
2023-10-21 20:56:13,428 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:56:15,469 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:56:15,469 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:56:15,716 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:56:16,707 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:56:17,731 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:56:17,731 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:56:17,944 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:56:19,993 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:56:19,993 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:56:20,225 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:56:21,845 DEBUG   HandlerThread:2540 [handler.py:handle_request():144] handle_request: status_report
2023-10-21 20:56:22,263 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:56:22,263 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:56:22,560 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:56:22,784 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\config.yaml
2023-10-21 20:56:22,786 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\output.log
2023-10-21 20:56:24,611 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:56:24,611 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:56:24,882 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-21 20:56:25,030 INFO    Thread-16 :2540 [dir_watcher.py:_on_file_modified():288] file/dir modified: e:\polar\pond_segmentation\wandb\run-20231021_205340-lph8yyg7\files\output.log
2023-10-21 20:56:26,887 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2023-10-21 20:56:26,890 ERROR   gpu       :2540 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
